src/history/hs.c

```diff
@@ -752,6 +752,20 @@ __wt_hs_insert_updates(WT_SESSION_IMPL *session, WT_PAGE *page, WT_MULTI *multi)
 
         prev_upd = upd = NULL;
 
+        /*
+         * Trim from the end until there is a full update. We need this if we are dealing with mixed
```

Comment:
Can you rename mixed mode operation to updates without timestamps? I'm using that term across this file.

Reply:
Fixed